scheduling/src/main/resources/META-INF/native-image/io.helidon.scheduling/helidon-scheduling/native-image.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Args=-H:IncludeResourceBundles=CronUtilsI18N
```

Comment:
please change to use of `resource-config.json`
```json
{
    "bundles": [
        {"name":"CronUtilsI18N"}
    ]
}
```
in the same location and remove this file.

Reply:
fixed